config/locales/devise.en.yml

```diff
@@ -43,13 +43,11 @@ en:
       invalid_token: 'The reset password token is invalid. Please try again.'
     registrations:
       start:
-        bullet_1: Please use this site to securely log in to your participating government agency.
+        bullet_1: login.gov is providing account services to %{app}.
```

Comment:
since you added an `app` var here, you need to pass that where we are using this `bullet_1` translation. (this is why the tests are broken on this branch)